yarn-project/types/src/tx_execution_request.ts

```diff
@@ -17,19 +18,23 @@ export class TxExecutionRequest {
     /**
      * Function arguments.
      */
-    public args: Fr[],
+    public argsHash: Fr,
     /**
      * Transaction context.
      */
     public txContext: TxContext,
+    /**
+     * Packed arguments
+     */
+    public packedArguments: PackedArguments[],
```

Comment:
It took me a while to understand why needed an _array_ of packed args here, instead of just the packed args that corresponded to the argsHash. Should we add a comment explaining it?